app/assets/templates/shared_components/form_buttons.html.haml

```diff
@@ -0,0 +1,45 @@
+.button-group
```

Comment:
Aah, this file is also never used and can be removed (we have the other (same) one in /static/...)